lib/pubid/iso/harmonized_stage_code.rb

```diff
@@ -50,7 +51,15 @@ class HarmonizedStageCode
       "95.60" => "Close of voting",
       "95.92" => "Decision not to withdraw International Standard",
       "95.99" => "Withdrawal of International Standard"
-    }
+    }.freeze
+
+    DRAFT_STAGES = %w[00.00 00.20 00.60 10.00 10.20 10.60 10.92 20.00 20.20 20.60 30.00
+                      30.20 30.60 30.92 40.00 40.20 40.60 40.92 40.93 50.00 50.20 50.60
+                      50.92].freeze
+
+    CANCELED_STAGES = %w[00.98 10.98 20.98 30.98 40.98 50.98 90.99 95.99].freeze
```

Comment:
90.99 is "Withdrawal of International Standard proposed by TC or SC", it is a "proposal" to withdraw, it has not been withdrawn. So it is still in published stage.

This is the only one to move to PUBLISHED_STAGES.